### DISCUSSION TITLE
替换为自定义图片资源文件

### DISCUSSION THREAD

**Discussion by AronnaxM** [PenUniverse/PenMods#253] [奇技淫巧]
本文测试设备为 P3 ( YDP038 CHN PRO )。
应该是比较简单的操作，仅涉及文件替换。很多人可能也发现了但一直没人提出来。
三代词典笔的静态图片资源文件 ~~应该~~ 都位于此路径下，二代笔没试过：

```
/oem/YoudaoDictPen/output/images
```

这里可以自定义你想要的图片，包括但不限于主界面卡片图标/背景色，状态栏、下拉菜单栏图标，设置图标之类的，不过务必先备份未修改的 images 文件夹以防万一。注意你替换的图片分辨率应与你替换的目标原始文件一致。在这里备份了三代笔的原始 images 文件夹，并且提供了一个将图标替换为 Material Design 风格的 images 文件：[AronnaxM/YDP-Res-Backup](https://github.com/AronnaxM/YDP-Res-Backup/releases/tag/Backup)

ADB的相关使用操作自行研究谢绝提问。

替换文件夹时没必要把原来的 images 文件夹删掉，可以只改个名字，只要保证有一个资源齐全的 images 文件夹就行，替换完重启词典笔主程序或者直接重启词典笔生效。

如果你找到了其他类似资源文件的位置欢迎在下面提出来。
也许未来可以写一个集成式的词典笔工具箱，把Penmod安装，数据库修改和资源文件替换之类的这些Discussion区提过的功能都加进去。

**Comment by Tone-cloud** [comment]
二代笔也是一样的

**Reply by AronnaxM** [reply]
路径可能确实是一样的，不过二代笔的背景图片大小和其他的一些资源跟三代笔应该是不一样的

**Reply by Tone-cloud** [reply]
二代有一些是要比三代小，有一些是一样的
